schemas/package.yml

```diff
@@ -72,3 +72,13 @@
 
         Use a short name, e.g. the license identifier from SPDX License List where possible (https://spdx.org/licenses/).
       example: Apache License 2.0
+
+    - name: type
+      level: extended
+      type: keyword
+      short: Package type
+      description: >
+        Type of package.
+
+        When installed from a package manager, this would contain the package manager name, e.g. RPM, DPKG, Homebrew, NPM.
+      example: RPM
```

Comment:
In order to guide in the right direction without getting into all of the subtleties discussed above, I think the following would work well enough:

```YAML
      description: >
        Type of package.

        This should contain the package file type, rather than the package
        manager name. Examples: rpm, dpkg, brew, npm, gem.
      example: rpm
```

Note also that I switched all examples to lowercase. This aligns better with the rest of ECS.